lib/xlat_tables/xlat_tables_common.c

```diff
@@ -277,11 +277,11 @@ static uint64_t mmap_desc(mmap_attr_t attr, unsigned long long addr_pa,
  * attributes of the innermost region that contains it. If there are partial
  * overlaps, it returns -1, as a smaller size is needed.
```

Comment:
Could you please update the function comment to better reflect its new behaviour?

I had an internal version of this patch sitting in our internal review system, so here's what I had written, but feel free to diverge from that:
```
/*
 * Look for the innermost region that contains the area at `base_va` with size
 * `size`. Populate *attr with the attributes of this region.
 *
 * On success, this function returns 0.
 * If there are partial overlaps (meaning that a smaller size is needed) or if
 * the region can't be found in the given area, it returns -1. In this case the
 * value pointed by attr should be ignored by the caller.
 */
```